lib/timeline/component/Group.js

```diff
@@ -503,9 +506,6 @@ class Group {
         forceRestack = this._didMarkerHeightChange.call(this) || forceRestack;
       },
 
-      // recalculate the height of the subgroups
-      this._updateSubGroupHeights.bind(this, margin),
```

Comment:
Removing this isn't necessary. Adding the above should have done the trick.

Reply:
I removed it just to avoid calling it twice. But I can add it back in.

Reply:
I think you should. Just to avoid special cases

Reply:
I think you should. Just to avoid special cases

Reply:
I think you should. Just to avoid special cases